README.md

```diff
@@ -70,8 +70,10 @@ From the `api/` directory:
 ./project.rb dev-up
 ```
 
-When the console displays "Dev App Server is now running", you can hit your
```

Comment:
This text isn't prominent, and there are about 30 more lines of output.  "Listening for socket" is the last line I see when dev-up is successful.